reddit_example.py

```diff
@@ -0,0 +1,15 @@
+import config
+import os
+import requests
+from pprint import pprint
+
+TOKEN = os.environ.get("REDDIT_ACCESS_TOKEN")
+
+# add authorization to our headers dictionary
+headers = {"User-Agent": "DangusBot/0.0.1", **{"Authorization": f"bearer {TOKEN}"}}
+
+# # while the token is valid (~2 hours) we just add headers=headers to our requests
+res = requests.get("https://oauth.reddit.com/api/v1/me", headers=headers)
+
+pprint("GET https://oauth.reddit.com/api/v1/me result:")
+pprint(res.json())
```

Comment:
This is an example file that you should be able to run with `python reddit_example.py` after getting the access token using the other file and adding it to your `.env` file